graphx/src/main/scala/org/apache/spark/graphx/VertexRDD.scala

```diff
@@ -128,7 +128,7 @@ abstract class VertexRDD[VD](
    *
    * @param other the other RDD[(VertexId, VD)] with which to diff against.
    */
-  def diff(other: RDD[(VertexId, VD)]): VertexRDD[VD]
+  def diff(other: RDD[(VertexId, VD)]): VertexRDD[VD] = ???
```

Comment:
We shouldn't put ??? here, since it means "we will implement this but we haven't got around to implement it yet".